package.json

```diff
@@ -55,6 +56,7 @@
         "tsConfigFile": "tsconfig.json"
       }
     },
-    "testRegex": "(/src/.*(\\.|/)(test|spec))\\.(jsx?|tsx?)$"
+    "testRegex": "(/src/.*(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
+    "testURL": "http://localhost"
```

Comment:
this is defined here but I don't see it used anywhere in the PR

Reply:
The jest tests wouldn't run without it on Ubuntu 16.04. Dunno. I believe the Bitshares PR had to add it as well. 

` SecurityError: localStorage is not available for opaque origins`

That's the error I get if I remove that line.